### PR TITLE
Add in meter and all associated styles

### PR DESCRIPTION
**Pull request by antelopeb**
Meter CSS and doc pages.

### CODE REVIEW

**Comment by umahaea** [comment]
@antelopeb - quick question. Since I see there some code changes made to elements sdk, as part of this PR, is the QA from your team writing the automated tests for it?

**Comment by StommePoes** [comment]
@antelopeb it may need to be made clearer in the docs, and I think I didn't mention it enough in my walk-through, but instead of writing tests in Elements itself (we used to), all tests are now run in Selenium.

So when something like a new icon is added, our QA has to be told the new unicode code point as the tests just check that. For adding new styles for any existing components, we need to list the styles that have changes and on which component so QA here knows where to make changes and which ones.
Finally for a completely new component, if the styles are only affecting this new component, then QA would test for these new styles and measure how they display in our supported (also mobile) browsers.

If you already have your own QA team, then they can write these tests and if they okay your code, we'll accept it.

**Comment by antelopeb** [comment]
@umahaea and @StommePoes - we don't have a QA team, so no, we won't be providing any Selenium tests.

Totally new component, and all CSS is directed at just that component, so no other changes need to be tested.

Is there anything else here I need to do to get this accepted?

**Comment by umahaea** [comment]
@antelopeb - you would also need to squash your series of commits into a single commit. The commit message has to be changed to "feat(progressbar): Add in progressbar and all associated styles", if not we will not see this message in the change logs. Also the QA tests will be reviewed and merged by PDA team only.

**Comment by StommePoes** [comment]
These steps for the squashing https://github.com/Pearson-Higher-Ed/docs/blob/master/origami-contributions.md which also has the link to teh Neo page with more explicit git commands if needed.

**Comment by antelopeb** [comment]
Sorry, I'm confused now. Do I need to delete this pull request, re-do my commit so it's one single commit, and then do another pull-request?

@StommePoes - I'm still not getting how to squash my commits. The rebase always seems to take place before hand. I didn't have any changes to merge when I pulled, and I'm not honestly sure why it created that merge commit anyways. Do I have to rebase instead of pull all the time to avoid it then!?

**Comment by StommePoes** [comment]
typo "progress bar's" -> progress bars (plural not possessive)

I will have to investigate the div inside the progress, because for browsers who recognise both, we may be breaking this spec rule:
```
Content model:
Phrasing content, but there must be no progress element descendants.
```
where, because the progressbar element itself has a role of progressbar, and then has a div inside also with a role of progressbar, this may not be kosher. I will test it out.

**Comment by antelopeb** [comment]
Commits squashed. I'll wait for @StommePoes testing before I make the change to the typo and rebase it in.

**Comment by umahaea** [comment]
@antelopeb - I will write the selenium tests pointing to this feature branch. I will reach out to you for any questions. Until we have the tests written, we will not merge this branch to V0.

File: scss/_meter.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
+progress[value] {
+	appearance: none;
+	-webkit-appearance: none;
+	-moz-appearance: none;
+	border: $pe-progressbar-border;
+	display: block;
+	width: 100%;
+	height: 20px;
+	color: $pe-progressbar-value; // for IE10
```
Comment (StommePoes) [inline scss/_meter.scss]: 10+, I think this also is true for 11 which we do support

**Comment by StommePoes** [comment]
This looks ok to me. I'm still trying to find out where aria-roledescription is as far as specs, but I see no harm in leaving it in and waiting for approval or support to kick in at some later date as well. It otherwise looks ok.

**Comment by StommePoes** [comment]
Looks like both Orca and VO screen readers have implemented role-description so I'm happy there.

**Comment by antelopeb** [comment]
@StommePoes - fixed your note on IE10+

**Comment by antelopeb** [comment]
@StommePoes I can't get the "out of date" message to go away... merged it in, rebased and squashed, but it still keeps telling me that it's out of date. Is this okay?

**Comment by StommePoes** [comment]
I'm getting it on Github and dunno why... 

I go to v0, pull latest, then switch back to my feature, rebase -i v0, then my feature branch should be up to date when I do a new force-push PR...

File: scss/_meter.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	height: 20px;
+	.pe-progressbar-inner {
+		position: absolute;
+		top: 0;
+		left: 0;
+		height: 100%;
+		background: $pe-progressbar-value;
+	}
+}
+.pe-progressbar-label {
```
Comment (umahaea) [inline scss/_meter.scss]: @antelopeb - The font-size and line-height here seems to be hard coded.. It should use pe-lable of typography(basic label).

**Comment by StommePoes** [comment]
New changes:
removed position:block
moved the vendor prefixes to a separate mixin though I hope in the future we won't have a linter that cares
reconfigured the inner div to hold the outer one open instead of absolute-positioning due to Chrome/Mac issue:
mac-chrome showing an extra 1px room under the inner div

**Comment by antelopeb** [comment]
@umahaea @StommePoes - I got a JIRA comment on this about font sizing. Do I need to fix those and get them in to have this merged? And the commits are up to 6, do those need to be packed into one commit again?

**Comment by umahaea** [comment]
@antelopeb - I had written that jira comment on the font sizing. Mallory has already incorporated those changes in this PR

**Comment by antelopeb** [comment]
@umahaea Great, thanks!

**Pull request merged**